polaris/polaris/utils.py

```diff
@@ -349,3 +350,15 @@ def validate_patch_request_fields(fields: Dict, transaction: Transaction):
                     gettext("missing %(field)s in %(category)s")
                     % {"field": field, "category": category}
                 )
+
+
+def to_decimals(float_value: float, precision: int) -> str:
+    """
+    Converting from float to string with the decimal precision.
+    """
+    return str(
+        decimal.Decimal(str(float_value)).quantize(
+            decimal.Decimal(".{}1".format("0" * (precision - 1))),
+            rounding=decimal.ROUND_05UP,
+        )
+    )
```

Comment:
I'm not sure we should provide this as a utility function. It has the assumption that `float_value` is less than 0 for example, so it seems pretty specific to a particular use case.

I also see most of the calls to this function casting their `float_value` parameter, only to have it cast back to a string here.

Have you seen how Polaris handles this logic in other parts of the code?

```
                transaction.amount_out = round(
                    transaction.amount_in - transaction.amount_fee,
                    transaction.asset.significant_decimals,
                )
```

Where `amount_in`, `amount_out`, and `amount_fee` are `Decimal` instances. I would avoid working with floats in general.